### PR TITLE
Fix Report Variables CSV Ordering

### PR DESCRIPTION
**Pull request by mitchute**
This PR addresses #6875 and also corrects the current 9.2->9.3 Report Variables CSV file. The following line in the CSV file must come __AFTER__ the modified report variables, otherwise, the last entry in the CSV will be skipped.

```old variable name,new variable name,-- add variable names (before this line) and leave off units -- <DELETE> to delete```

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/Transition/SupportFiles/Report Variables 9-2-0 to 9-3-0.csv [hunk 1]
```diff
@@ -1,6 +1,5 @@
 9.2.0,9.3.0,Transition notes
 12,12,These numbers should be the number of report variables in the following list (including deletes). Two columns/numbers.
-old variable name,new variable name,-- add variable names (before this line) and leave off units -- <DELETE> to delete
 Water to Water Heat Pump Load Side Heat Transfer Rate,Heat Pump Load Side Heat Transfer Rate,
 Water to Water Heat Pump Load Side Heat Transfer Energy,EIR Heat Pump Load Side Heat Transfer Energy,
```
Comment (mjwitte) [inline src/Transition/SupportFiles/Report Variables 9-2-0 to 9-3-0.csv]: @mitchute These same changes are also in #7448 but it's better that these get fixed in a separate PR for better visibility. After this merges, I'll resolved conflicts over there.

Note that this line also needs a fix, delete EIR. 
~EIR~ Heat Pump Load Side Heat Transfer Energy

Comment (mitchute) [reply src/Transition/SupportFiles/Report Variables 9-2-0 to 9-3-0.csv]: @mjwitte Thanks for pointing that out. I'll make the changes soon.

**Comment by mitchute** [comment]
@mjwitte this PR is ready.

**Review by mjwitte — COMMENTED** [review]
Tested the 8-8-0 to 8-9-0 transition and verified the fix.

The Water to Water Heat Pump variables are a problem, because the old names in this group are still in use by other heat pumps (e.g. HeatPump:WaterToWater:ParameterEstimation:* and HeatPump:WaterToWater:EquationFit:*). Transition doesn't distinguish output variable names across component types. So, you'll either need to add some magic to transition (which would be difficult to discern if an incoming idf had more than one type of heat pump) or change the outputs for the other heat pumps to match these?  

I've posted a new issue #7733 to resolve this, but I'm going to merge this PR so it doesn't block #7448.

**Pull request merged**